src/main.ts

```diff
@@ -3,23 +3,34 @@ import Vuex from 'vuex'
 import App from './App.vue'
 import router from './router'
 
-import "bootstrap/dist/css/bootstrap.css";
-import "bootstrap/dist/css/bootstrap-theme.css";
-
 Vue.use(Vuex)
 
 Vue.config.productionTip = false
 
-new Vue({
-  router,
-  store: new Vuex.Store({}),
-  render: (h) => h(App),
-}).$mount('#app')
-
-// required inline due to front-loading of imports and having jQuery dependency
-window.onload = function () {
-  import('jquery')
-    .then((jQuery) => {
-      import('bootstrap')
-    })
+function exportAsGlobal(exports) {
+  Object.assign(window, exports)
+}
+
+async function initializeBootstrapFramework() {
+  import('bootstrap/dist/css/bootstrap.css')
+  import('bootstrap/dist/css/bootstrap-theme.css')
+
+  const {default: ImportedJquery} = await import('jquery')
+  exportAsGlobal({
+    $: ImportedJquery,
+    jQuery: ImportedJquery})
+
+  await import('bootstrap')
```

Comment:
Hi @jcbashdown, I did some testing using DevTools' throttling and was still able to reproduce. I've opted to push these guts into an async function to handle the dependencies more explicitly.